frontend/app/src/entities/events/ui/global-filter.tsx

```diff
@@ -0,0 +1,108 @@
+import { AttributeSchema, RelationshipSchema } from "@/entities/schema/types";
+import { focusVisibleStyle } from "@/shared/components/style-rac";
+import { Popover, PopoverContent, PopoverTrigger } from "@/shared/components/ui/popover";
+import useFilters from "@/shared/hooks/useFilters";
+import { classNames } from "@/shared/utils/common";
+import { Icon } from "@iconify-icon/react";
+import React, { useState } from "react";
+import { Tag, TagProps } from "react-aria-components";
+import { GlobalFilterForm } from "./global-filter-form";
+
+interface FilterTagProps extends TagProps {
+  label: React.ReactNode;
+  name: string;
+  fieldSchema: AttributeSchema | RelationshipSchema;
+}
+
+export function GlobalFilter({ label, name, fieldSchema, ...props }: FilterTagProps) {
+  const [filters, setFilters] = useFilters();
+  const [showFilters, setShowFilters] = useState(false);
+
+  const currentFilter = filters.find((filter) => filter.name.startsWith(name));
+
+  const handleRemoveFilter = (filterName: string) => {
+    setFilters(filters.filter((f) => f.name !== `${filterName}__value`));
+  };
+
+  const getFilterDisplayValue = () => {
+    if (typeof currentFilter?.value === "boolean") {
+      return JSON.stringify(currentFilter?.value);
+    }
+
+    if (fieldSchema.kind === "Dropdown") {
+      return fieldSchema.choices?.find((choice) => {
+        return choice.name === currentFilter?.value;
+      })?.label;
+    }
+
+    if (Array.isArray(currentFilter?.value)) {
+      return currentFilter?.value
+        .map((value) => {
+          return value.display_label;
+        })
+        .join(", ");
+    }
+
+    return currentFilter?.value;
+  };
+
+  return (
+    <Tag
+      className={classNames(
+        focusVisibleStyle,
+        "group text-gray-600 text-sm whitespace-nowrap bg-neutral-100 rounded-full inline-flex items-center gap-1.5 pl-1 border border-gray-300 cursor-pointer",
+        "data-[hovered]:bg-gray-100 data-[hovered]:border-custom-blue-700"
+      )}
+      textValue={`${label} contains ${currentFilter?.value}`}
+      {...props}
+    >
```

Comment:
when filter is unset, I thing we should use the same style as FilterSuggestionTag. To make UI consistent accross the app